src/elements/content-sidebar/Sidebar.js

```diff
@@ -175,6 +182,46 @@ class Sidebar extends React.Component<Props, State> {
         }
     }
 
+    /**
+     * Handle keyboard interaction
+     *
+     * @param {Array<string>} tablist - Contains the pathnames of tabs
+     * @param {SyntheticKeyboardEvent} event - Keyboard event
+     */
+    onKeyDown = (tablist: Array<string>) => (event: SyntheticKeyboardEvent<>): void => {
+        const { history } = this.props;
+        const { nativeEvent } = event;
+
+        nativeEvent.stopImmediatePropagation();
```

Comment:
Is native stuff needed? Dont see use of addeventlistener

Reply:
not needed...